test/integration/serving/serving_test.go

```diff
@@ -289,3 +292,141 @@ func fakeCloudProviderFactory(io.Reader) (cloudprovider.Interface, error) {
 		DisableRoutes: true, // disable routes for server tests, otherwise --cluster-cidr is required
 	}, nil
 }
+
+func TestControllerManagerServingStatusz(t *testing.T) {
+
+	// authenticate to apiserver via bearer token
+	token := "flwqkenfjasasdfmwerasd" // Fake token for testing.
```

Comment:
Do we need the token setup too?

Reply:
Yes, it turns out api server kube-config is needed to be used to create kube-controller-manager.